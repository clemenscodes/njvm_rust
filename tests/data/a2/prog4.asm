//
// prog4.asm -- an assembler example with global variables
//
