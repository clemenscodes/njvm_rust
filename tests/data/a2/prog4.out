Ninja Virtual Machine started
000:	asf	4
001:	pushc	-6
002:	popl	0
003:	pushc	2
004:	popl	1
005:	pushc	4
006:	popl	2
007:	pushl	0
008:	pushl	1
009:	pushl	2
010:	add
011:	add
012:	popl	3
013:	pushl	3
014:	wrint
015:	pushc	10
016:	wrchr
017:	rsf
018:	halt
0
Ninja Virtual Machine stopped
